webpack.config.js

```diff
@@ -25,6 +25,9 @@ module.exports = {
     bootstrap: './assets/src/scss/bootstrap-build.scss',
     "country-selector-old": './assets/src/scss/partials/country-selector-old.scss',
     "country-selector-new": './assets/src/scss/partials/country-selector-new.scss',
+    "navigation-bar-old": './assets/src/scss/layout/_navbar-old.scss',
+    "navigation-bar-dark": './assets/src/scss/partials/navigation-bar-dark.scss',
+    "navigation-bar-light": './assets/src/scss/partials/navigation-bar-light.scss',
```

Comment:
As an alternative approach, [using inline styles](https://github.com/greenpeace/planet4-master-theme/blob/40aac328a327e3565c38232c0a3604192af081b0/templates/css-variables.twig#L1-L7) to set different root properties has a few advantages.

- No need to build an additional CSS file for each style.
- It's generally recommended to load critical CSS inline.
- Easier to make compatible with themes on campaign pages, which already work that way.

But we would need to use a JSON object instead, or a PHP array, for the key value pairs. It's fine for now to go with the separate CSS files :+1:

Reply:
:ok_hand:  I'll be fine with anything that put those rules in explicitely named files, as long as there's a comment somewhere pointing at where they come from; finding the origin of inline styles can be really painful, especially if the property names are generated programmatically